### ISSUE TITLE
Doesn't control client

### ISSUE THREAD

**Issue by Inrego** [Hellowlol/bw_plex#118]
I've just set this up with docker, but I'm having an issue. It seems to recognize the playing media just find, and also find the intro end timestamp.. But the client doesn't skip to the end. Am I missing something?

```
2020-02-17 00:34:32,708 :: bw_plex :: DEBUG :: plex.py:1191 :: Found Dexter.s03e07 theme start 01:42, theme end 03:14, ffmpeg_end 03:18 progress 00:00 fake_progress 00:10 best_time 03:14 credits_start 00:00 credits_end 00:00
2020-02-17 00:34:39,414 :: bw_plex :: DEBUG :: plex.py:1191 :: Found Dexter.s03e07 theme start 01:42, theme end 03:14, ffmpeg_end 03:18 progress 00:06 fake_progress 00:16 best_time 03:14 credits_start 00:00 credits_end 00:00
2020-02-17 00:34:49,413 :: bw_plex :: DEBUG :: plex.py:1191 :: Found Dexter.s03e07 theme start 01:42, theme end 03:14, ffmpeg_end 03:18 progress 00:16 fake_progress 00:26 best_time 03:14 credits_start 00:00 credits_end 00:00
2020-02-17 00:34:59,410 :: bw_plex :: DEBUG :: plex.py:1191 :: Found Dexter.s03e07 theme start 01:42, theme end 03:14, ffmpeg_end 03:18 progress 00:26 fake_progress 00:36 best_time 03:14 credits_start 00:00 credits_end 00:00
```

Here's my docker command: `docker run --rm --name=bw_plex -e PUID=1000 -e PGID=1000 -e TZ=Europe/Berlin -e url=http://IP_ADDRESS:32400/ -e token=TOKEN -v /opt/bw_plex:/config -v /gmedia:/mnt/unionfs/
Media hellowlol/bw_plex`

**Comment by Hellowlol** [comment]
I dont know the reason. I need the entire log not just a snippet. The only think i can guess based on the log you provided is that the playback hadn’t reached the point where the client seeks to the end of the intro. The file will play as normal until the start of theme music then seek to the end. It also requires that the client is controllable (on same local network)

**Comment by Inrego** [comment]
You were on the right track. The intro didn't start yet. Now I'll set it up to skip previously as well.

When using the Plex app for Windows 10, bw_plex times out when trying to skip the intro. I am guessing it's due to firewall. Is there any info on what ports need to be open in order for it to work?

**Comment by Hellowlol** [comment]
🤷‍♂️ the client support are flaky. I’m using this on iOS mainly. If plexapi supports the client it should work. If you know some python you could write a test script that just tests the seek command to try to debug it. So see the wiki for clients support.

**Comment by Inrego** [comment]
I'm a C# veteran, but I don't rly have any python experience.

I've tested with Plex app on Android TV (Nvidia SHIELD). I've also tested with Plex app on my Android phone. Both worked great, so I guess you can add those to your supported list.
I also tested on Chromecast which also worked great. I was casting from Android phone if that makes any difference.

Getting it working on Windows Plex app is not a high priority for me, but I guess for the sake of functionality of bw_plex it'd be worth checking out. I'm willing to help in any way you need.
From the Plex app on my Android phone, I am able to connect to the Windows plex app and remote control it. So it does support remote control of some sort. I'm pretty sure it's related to this setting:
![image](https://user-images.githubusercontent.com/2242329/74720106-8701ae00-5235-11ea-8549-5dd42c86a173.png)

Basically, it looks pretty much like it's just a wrapper of the web app. The web app also have this same setting, so I also tested it here, and bw_plex doesn't work here either. I guess it's worth noting that remote controlling web app from Android App doesn't seem to work properly either (although I have seen it working before a long time go, maybe it's because I've just remote controlled Plex app on same machine).
